pipeline/bootstrap-pipeline.sh

```diff
@@ -64,7 +64,7 @@ helm_image=
 build_image=
 if [[ "${github}" == "github.com" ]]; then
     is_public_repo="true"
-    build_image="docker.io/yakinikku/suede_compile"
+    build_image="docker.io/yakinikku/suede_compile:latest"
```

Comment:
Can we replace the `latest` tag with the image digest?
`build_image="docker.io/yakinikku/suede_compile:@sha256:c7643deca1dd36c0a9c4aa1cc7315167a773da5bbff0597bb6630d7e013867f3`